Sources/ProjectDescription/HelpersLocation.swift

```diff
@@ -0,0 +1,119 @@
+import Foundation
+
+/// The location to a directory containing sources for `ProjectDescriptionHelpers`.
+public struct HelpersLocation: Codable, Equatable {
+
+    /// The type of location `local` or `git`.
+    public let type: LocationType
+
+    /// A `Path` to a directory containing `ProjectDescriptionHelpers`.
+    ///
+    /// Example:
+    /// ```
+    /// .local(path: "/User/local/bin")
+    /// ```
+    public static func local(path: Path) -> Self {
```

Comment:
Tuist aims for convention over configuration and I think this is a good example where the convention is useful. I think the opinion on where helpers should leave should be Tuist's. The more decisions that Tuist makes the fewer users have to make and thus focus on the important things. Having said that, I'd turn this into:

```swift
public var local: Self {}
```

Reply:
This is a location to where your reused location helpers are, not where they should be saved, so I believe it should stay as-is

Reply:
Correct, this was intended so that you could store helpers somewhere on your machine locally. This doesn't determine where they will be copied to.

Reply:
Having said that however, if we dont want to maintain the code required for local helpers/plugins and would rather just default to remote that works for me as well.

Reply:
I think it is useful during development of the remote helpers to be able to have a local version (instead of having to constantly push your code), so I'm in favor of keeping it